### ISSUE TITLE
Custom resource

### ISSUE THREAD

**Issue by spuder** [evertrue/exhibitor-cookbook#44]
It would be nice if this cookbook was converted to use a custom resource so it is more consistent with the zookeeper cookbook

**Comment by jeffbyrnes** [comment]
@spuder indeed! I’ve no bandwidth at the moment to work on this, unfortunately.

Pull requests are welcome.
